src/compare-tree/key-path.js

```diff
@@ -0,0 +1,40 @@
+const KEY_DELIMETER  = '.'
+const PATH_DELIMETER = ','
```

Comment:
ah, looks like the `'.'` convention is already defined in this project?https://github.com/vigetlabs/microcosm/blob/93568b41a3a6e3cf55474d645eca6d79d9ae64a6/src/utils.js#L4